src/ApartmentMigration.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace BuildEmpire\Apartment;
+
+use Illuminate\Database\Migrations\Migration;
+
+class ApartmentMigration extends Migration
+{
+    protected $schemas = [];
+    protected $fileName = false;
+    protected $originalTable;
+
+    /**
+     * ApartmentMigration constructor.
+     */
+    public function __construct()
+    {
+        $schema = new Schema();
+        $this->schemas = $schema->getAllSchemas();
+        $this->fileName = $this->getMigrationFileName();
+        $this->originalTable = $this->table;
+    }
+
+    /**
+     * Run/Up all apartment migrations.
+     *
+     * @param bool $ranViaArtisan
+     */
+    public function up($ranViaArtisan = true) {
+        foreach($this->schemas as $schema) {
+
+            $schemaTablePrefix = $schema->name . '.';
```

Comment:
Surely the `dot` should get appended within the `setSchemaTable` method - I don't think we should expect the `schemaTablePrefix` to include a dot.